docs/UserGuide.adoc

```diff
@@ -123,50 +127,260 @@ Format 2: `edit INDEX`
 You can omit the (optional) parameters by leaving it empty. If all parameters are empty, no edits will occur.
 
 Examples:
-* `edit 9` +
-  `Please enter the name of the expense to be updated (optional):` +
-  `Please enter the cost of the expense to be updated (optional): 450` +
-  `Please enter the category of the expense to be updated (optional): food` +
-  `Please enter the date of the expense to be updated (optional):` +
-  `Please enter the explanation of the expense to be updated (optional):`
+`edit 9` +
+`Please enter the name of the expense to be updated (optional):` +
+`Please enter the cost of the expense to be updated (optional): 450` +
+`Please enter the category of the expense to be updated (optional): food` +
+`Please enter the date of the expense to be updated (optional):` +
+`Please enter the explanation of the expense to be updated (optional):`
 
 
 === Deleting an expense : `delete`
 
 Deletes the specified expense from the finance tracker. +
 Format: `delete INDEX`
 
-****
+[NOTE]
+====
 * Deletes the expense at the specified `INDEX`.
 * The index refers to the index number shown in the displayed expense list.
 * The index *must be a positive integer* 1, 2, 3, ...
-****
+====
 
 Examples:
 
 * `list` +
 `delete 2` +
 Deletes the 2nd expense in the finance tracker.
 
-=== Selecting a person : `select`
+=== Add a budget : `addBudget`
+
+Adds a budget with a time frame to the tracker. +
+Format 1: `addBudget $/AMOUNT c/CATEGORY sd/START_DATE ed/END_DATE`
+
+[NOTE]
+====
+* The categories include: “food”, “shopping”, “work”, “transport”, “utilities”, “healthcare”,
+“entertainment” and “others” which are case insensitive.
+* Note that the program only limits one budget for each category.
+====
+
+Examples:
 
-Selects the person identified by the index number used in the displayed person list. +
-Format: `select INDEX`
+* `addBudget c/food $/400  sd/1-2-2019 ed/28-2-2019`
+* `addBudget c/all $/12000 sd/1-1-2019 ed/31-12-2019`
 
-****
-* Selects the person and loads the Google search page the person at the specified `INDEX`.
-* The index refers to the index number shown in the displayed person list.
-* The index *must be a positive integer* `1, 2, 3, ...`
-****
+Format 2: `addBudget`
 
 Examples:
 
-* `list` +
-`select 2` +
-Selects the 2nd person in the address book.
-* `find Betsy` +
-`select 1` +
-Selects the 1st person in the results of the `find` command.
+* `addBudget` +
+`Please enter the category of the budget: food` +
+`Please enter the cost of the budget: 400` +
+`Please enter the start date of budget: 1-2-2019` +
+`Please enter the end date of budget: 28-2-2019`
+
+=== Edit a budget: `editBudget`
+
+Edits a budget in the finance tracker.
+
+Format 1: `editBudget c/CATEGORY [$/AMOUNT] [sd/NEW_START_DATE] [ed/NEW_END_DATE]`
+
+Examples:
+
+* `editBudget c/food $/500`
+* `editBudget c/all ed/31-3-2019`
+* `editBudget c/all $/5000 sd/1-1-2019 ed/31-3-2019`
+
+Format 2: `editBudget`
+
+Examples:
+
```

Comment:
Suggest to add the following line from 'edit' into 'editBudget' as it is applicable as well. 

"You can omit the (optional) parameters by leaving it empty. If all parameters are empty, no edits will occur."